src/modules/launcher/PowerLauncher.UI/App.xaml

```diff
@@ -12,23 +12,17 @@
             </ResourceDictionary.MergedDictionaries>
             <ResourceDictionary.ThemeDictionaries>
                 <ResourceDictionary x:Key="Light">
+                    <Color x:Key="BackgroundColor">#FFf2f1f2</Color>
                     <SolidColorBrush x:Key="PrimaryTextColor" Color="Black"/>
-                    <SolidColorBrush x:Key="SystemChromeLow" Color="{ThemeResource SystemChromeLowColor}" />
-                    <AcrylicBrush x:Key="BackdropAcrylicBrush" BackgroundSource="Backdrop" TintColor="White" TintOpacity="0.9" FallbackColor="White"/>
-                    <AcrylicBrush x:Key="KeyboardShortcutAcrylicBrush" BackgroundSource="Backdrop" TintColor="White" TintOpacity="0.9" FallbackColor="White"/>
-                    <SolidColorBrush x:Key="ItemBackgroundColor" Color="#FFFFFFFF"/>
-                    <Color x:Key="DropShadowColor">#FF979797</Color>
-                    <x:Double x:Key="DropShadowBlurRadius">8</x:Double>
+                    <SolidColorBrush x:Key="SystemChromeLow" Color="{StaticResource BackgroundColor}" />
+                    <AcrylicBrush x:Key="BackdropAcrylicBrush" BackgroundSource="Backdrop" TintColor="{StaticResource BackgroundColor}" TintOpacity="0.9" FallbackColor="{StaticResource BackgroundColor}"/>
```

Comment:
Will changing the BackgroundSource to "HostBackdrop" have any effect??

Reply:
Or maybe to SystemControlAcrylicWindowBrush ??

Reply:
*0.9* `TintOpacity` is a lot. I believe 0.6 or 0.7 is a value that can make the acrylic look better. And I think `BackgroundSource` should be `HostBackdrop`.

Reply:
```suggestion
                    <AcrylicBrush x:Key="BackdropAcrylicBrush" BackgroundSource="HostBackdrop" TintColor="{StaticResource BackgroundColor}" TintOpacity="0.6" FallbackColor="{StaticResource BackgroundColor}"/>
```

Reply:
Also, `RevealBrush` would make the suggestions look better (like a combobox, which uses both acrylic and reveal brushes and the `SystemAccentColor` to highlight the selected option)

Reply:
@SFM61319 Please note that this branch was using UWP/XAML Islands.. This was giving performance issues however, so we're back to using WPF. Therefore, this PR can be ignored and is outdated.

Reply:
@niels9001 but wouldn't that just add to the existing UI inconsistency across win10?

Reply:
With the current implementation of Run, we try to mimic WinUI as much as possible.

It's just the current state of the technology. Hopefully we can move this over to WinUI at some point so we can have access to e.g. Acrylic and Reveal.